docs/source/optimizations/ibm/ibmq_compile_qss.ipynb

```diff
@@ -21,9 +21,37 @@
     "</a>"
```

Comment:
<p>Following comment from the css notebook:</p><pre class="ql-syntax" spellcheck="false">Line #1.    # Visualize the pulse sequence for the second compiled circuit
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/539/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
Fixed.